packages/flutter/lib/src/painting/strut_style.dart

```diff
@@ -442,6 +495,9 @@ class StrutStyle extends Diagnosticable {
     );
   }
 
+  @override
+  String toStringShort() => '$runtimeType';
```

Comment:
Is there a reason that this is the desired behavior?

Reply:
This was based around the same override in `text_style.dart:935` where presumably, the full toString that includes all of the properties, is often times too long and makes output unreadable. A similar argument can be applied to StrutStyle, which also contains a large number of potentially cluttering properties that are not necessarily relevant in the short version of the toString.